azurerm/resource_arm_role_assignment.go

```diff
@@ -163,6 +176,7 @@ func resourceArmRoleAssignmentRead(d *schema.ResourceData, meta interface{}) err
 		d.Set("scope", props.Scope)
 		d.Set("role_definition_id", props.RoleDefinitionID)
 		d.Set("principal_id", props.PrincipalID)
+		d.Set("skip_service_principal_aad_check", d.Get("skip_service_principal_aad_check").(bool))
```

Comment:
since we're just reading this value back in we can remove this field (and add it to the ignore list during the import tests)

Reply:
Sure thing.